evergreen/config.yml

```diff
@@ -592,7 +592,7 @@ tasks:
                 export DEVELOPER_DIR="${xcode_developer_dir}"
             fi
 
-            ./evergreen/install_baas.sh -w ./baas-work-dir -b 9398abf62ad6876c38cc675bce47b4f96786a1d6 2>&1 | tee install_baas_output.log
+            ./evergreen/install_baas.sh -w ./baas-work-dir -b master 2>&1 | tee install_baas_output.log
```

Comment:
Do we want to use `master` or the released version? Or do you think it matters? IIRC, we switched to master since there was some feature we needed that hadn't been released yet. Probably couldn't hurt to use the latest master though...
```
./evergreen/install_baas.sh -w ./baas-work-dir 2>&1 | tee install_baas_output.log
```

Reply:
I want to use master, which is why I changed it to master.